src/common/Tile/GenreSection/index.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
+import { useSelector } from "react-redux";
 import GenreTile from "../GenreTile";
 import { StyledSection } from "./styled";
+import { selectGenres } from "../../../features/movies/moviesSlice"
 
-const GenreSection = ({ horizontal, genres }) => (
-  <StyledSection horizontal={horizontal}>
-    {genres.map((genre) => (
-      <GenreTile key={genre} horizontal={horizontal} genre={genre} />
-    ))}
-  </StyledSection>
-);
+const GenreSection = ({ horizontal, genres }) => {
+
+  const genresList = useSelector(selectGenres);
+
+  return (
+    <StyledSection horizontal={horizontal}>
+      {genresList.genres && genres?.map((genre) =>
+        <GenreTile
+          key={genre}
+          horizontal={horizontal}
+          genre={
+            genresList.genres[(genresList.genres?.findIndex(({ id }) => id === genre))].name
```

Comment:
ja bym to zrobiła za pomocą filter `genresList.genres.filter(({ id }) => id === genre)[0].name`
wydaje mi się bardziej eleganckie, ale tak też jest spoko

Reply:
Tutaj się nie zgodzę... 
akurat nie podoba mi się, fakt, że na sztywno jest wbite "od indexu 0", zawsze staram się unikać w logice sztywnych wartości a moja propozycja niezależnie od indexu dopasuje id z genres_ids do nazwy

Reply:
po wyfiltrowaniu tablica powinna mieć tylko jeden element, więc 0 to jedyny index, ale rozumiem :)
Swoją drogą żadne z tych rozwiązań nie jest odporne na przypadek gdy w tablicy genres nie ma gatunku o szukanym id, ale może załóżmy że api nie ma błędów i taka sytuacja się nie wydarzy 🙈

Reply:
Na youcode się projekt nie kończy, dopisz proszę walidację tej sprawy do zadań, później pewnie będzie okazja się temu przyjrzeć